README.md

```diff
@@ -39,17 +39,17 @@ AdministratorAccess (AWS Managed)
 - To have your DNS domain.
 
 ### Overview
-<TBD>
+
+- TBD
 
 ## Deployment
 
-Add additional notes about how to deploy this on a live system
+1. Deploy `tf-init` folder
```

Comment:
Instead of having simple words, you can mention a bit clear steps so that we can make sense.

Reply:
Okay :)